src/main/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/ValidationModelService.java

```diff
@@ -71,8 +71,10 @@ public void addExistingRoleAssignments(AssignmentRequest assignmentRequest, List
 
         }
         for (String actorId : userIds) {
-            facts.addAll(persistenceService.getExistingRoleAssignment(UUID.fromString(actorId)));
-            facts.addAll(idamRoleService.getIdamRoleAssignmentsForActor(actorId));
+            if (actorId != null) {
```

Comment:
actor Id is a mandatory field, if it is null it should be captured at parse request side. we need not check it.